code/modules/antagonists/clockcult/clock_helpers/slab_abilities.dm

```diff
@@ -178,6 +178,10 @@
 	. = ..()
 
 /obj/item/projectile/kindle/on_hit(atom/target, blocked = FALSE)
+	kindle(target)
+	..()
+	
+/proc/kindle(target) //I HAVE NO IDEA WHAT I'M DOING
```

Comment:
help

Reply:
uh

Reply:
move the proc to kindleium and have the kindle projectile add kindleium.

Reply:
That would be a balance change, you could chug charcoal/antitox to significantly reduce stun time.

Reply:
and stun time reduction is a big deal because script reciting speed puts a fundamental limit on how fast they can pull you up to reebee or cuff you, and as-is you can only just barely get someone up to reebee and cuff them before it wears off.

Reply:
Oh, and unlike meth/whatever charcoal has no OD and is available en-mass in medbay so you could just chug hundreds of it at the first sign of cultists and get resistance for half of the round.

Reply:
ok how about livingcheck L.kindle() and at least localize the proc to mob/living.

a proc for literally everything that only does stuff for a particular group of mobs is silly.